src/View/User/Includes/_welcome_background_slideshow.html.twig

```diff
@@ -0,0 +1,13 @@
+{# User/Includes/_welcome_background_slideshow.html.twig #}
+
+<div class="slider">
+        <div class="load">
+        </div>
+        <div class="content">
+            <div class="principal">
+            <h1>lorem ipsum</h1>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+                {% include 'User/Includes/_welcome_card.html.twig' %}
```

Comment:
Is it really necessary to include another twig? the text must change?

Reply:
we created 2 distinct elements at the start in our user stories